spec/database_helpers.rb

```diff
@@ -0,0 +1,6 @@
+require './lib/database_connection'
+
+def persisted_data(id:, table:)
+  connection = PG.connect(dbname: 'chitter_test')
```

Comment:
Could have brought your database class into this? Not required, just means you have consistent use of it across the application